app/components/markdown/markdown_table/markdown_table.js

```diff
@@ -73,6 +74,38 @@ export default class MarkdownTable extends React.PureComponent {
         });
     };
 
+    renderPreviewRows = (drawExtraBorders = true) => {
+        const style = getStyleSheet(this.props.theme);
+
+        const tableStyle = [style.table];
+        if (drawExtraBorders) {
+            tableStyle.push(style.tableExtraBorders);
+        }
+
+        // Add an extra prop to the last row of the table so that it knows not to render a bottom border
+        // since the container should be rendering that
+        const maxColumns = Math.floor(this.props.deviceWidth / CELL_WIDTH);
```

Comment:
this looks alright, I just want to avoid using the redux value as it will eventually go away